src/core/str.js

```diff
@@ -10,10 +10,26 @@
 *     http://bunkat.github.com/pseudoloc
 */
 pseudoloc.str = function(str) {
+  function makeTokenRegex(delims, tokenNameDelim) {
+    var tokenMatchers = delims.reduce(function(result, delim) {
+      if (delim.hasOwnProperty('both')) {
+        result.push(delim.both + tokenNameDelim + delim.both);
+      } else if (delim.hasOwnProperty('start') && delim.hasOwnProperty('end')) {
+        result.push(delim.start + tokenNameDelim + delim.end);
+      } else if (delim.hasOwnProperty('full')) {
+        result.push(delim.full);
+      }
+      return result;
+    }, []);
+    return new RegExp(tokenMatchers.join('|'), 'g');
+  }
+
```

Comment:
This is the core of the change. Now it uses the `delimiters` array (which by default contains the values from `startDelimiter` and `endDelimiter`) to create the token regular expression.

It constructs a regular expression (as a string) for each element of the array, then joins them together with `|` ("or"). That becomes the full regular expression that is used to identify things to _not_ pseudolocalize.